### ISSUE TITLE
Canvas move when using Pencil Tool with touch screen

### ISSUE THREAD

**Issue by bth** [SVG-Edit/svgedit#394]
**Describe the bug**
When I tried to use "Pencil Tool" with my touch screen, canvas moves the same time line is drawn.
I can see this behavior on Google Chrome but not on Firefox.

**To Reproduce**
1. On Google Chrome
2. Select "Pencil Tool"
3. Touch screen to draw a line
4. Line is drawn, but the canvas moves at the same time

**Expected behavior**
I think canvas should not move when you try to draw a line by touch.

**SVG-Edit environment (IMPORTANT)**
- File for SVG-Edit: `svg-editor.html`
- Version: commit 623a8a715acf5408d04586ba134392287ec57d0e (current master)
- Protocol: file

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04.6 LTS
 - Browser: Google Chrome
 - Version:  80.0.3987.122 (64 bits)

**Screenshots**
![bug_svgedit](https://user-images.githubusercontent.com/2178086/76115747-08ed2780-5fe9-11ea-802f-0a3e1a63ae83.gif)

**Comment by bth** [comment]
When I reproduce the behavior, I got this message on Chrome's console: 
```
[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See  https://www.chromestatus.com/features/5093566007214080
touchHandler @ svgedit-config-iife.js:264
[Intervention] Ignored attempt to cancel a touchend event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchHandler @ svgedit-config-iife.js:264
```

**Comment by brettz9** [comment]
Just so you know, and as mentioned on the README, this is not a very actively maintained project. I'm accepting PRs and labeling issues, but only just to keep things alive in case others may come along to get things improved.

**Comment by brettz9** [comment]
If you are debugging yourself, I recommend using the svg-editor-es.html module version as you can work directly on the regular `editor` source files.

**Comment by bth** [comment]
Maybe it's similar to this issue (on jquery.nicescroll project) https://github.com/inuyaksa/jquery.nicescroll/issues/799

When I set `passive` attribute to `false` on events listeners (in `svgedit-config-iife.js`), I don't reproduce the behavior:
```javascript
document.addEventListener('touchstart', touchHandler, {capture : true, passive: false});
document.addEventListener('touchmove', touchHandler, {capture : true, passive: false});
document.addEventListener('touchend', touchHandler, {capture : true, passive: false});
document.addEventListener('touchcancel', touchHandler, {capture : true, passive: false});
```

**Comment by brettz9** [comment]
Yes, it looks like https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Improving_scrolling_performance_with_passive_listeners explains this well. 

However, I wonder if we can avoid doing this and instead see if we can listen for those touch events on something other than `window`, `document`, or `document.body` so we don't lose out on the performance fixing of scrolling that Chrome/Firefox do here (assuming it is possible). It may be though that we need these at a higher level than just the canvas so that toolbar buttons and such can also work correctly with touch events.

**Comment by jfhenon** [comment]
fixed with PR #634
